ghost/core/core/server/api/endpoints/themes.js

```diff
@@ -15,6 +16,18 @@ module.exports = {
         }
     },
 
+    active: {
```

Comment:
We are doing an "active" here instead of "read" because for the client it would be hard to find out which theme is currently active? We don't usually name controller functions by the "filter", so not sure if this naming is necessary?

Reply:
More because we only have cached data available for the active theme. But thinking about it further we could change it to a read action with a filter. It would just run a new gscan report on the theme if it is not asked.

Reply:
And it would throw a validation error if the requested theme is not loaded, but I think they need to be loaded and that is probably an edge case right now